forge/forge/op/eval/forge/eltwise_binary.py

```diff
@@ -225,14 +225,19 @@ def backward(op_type, attr, ac, operand, inputs, output, grad):
             for i in range(len(shapes[operand])):
                 if shapes[operand][i] < grad_shape[i]:
                     # Negative indexing for reduce axis
-                    grad = ac.op("reduce_sum", (grad,), (i - grad_shape_len,), {"keep_dim": True})
+                    grad = ac.op(
+                        "reduce_sum",
+                        (grad,),
+                        (i - grad_shape_len,),
+                        {"keep_dim": True, "dim_arg": [i - grad_shape_len]},
+                    )
         return ac.op(Nop.create(), (grad,))  # pass gradient through
 
     elif op_type == "subtract":
         if inputs[operand].shape != grad.shape:
             for i in range(len(shapes[operand])):
                 if shapes[operand][i] < grad.shape[i]:
-                    grad = ac.op("reduce_sum", (grad,), (i,), {"keep_dim": True})
+                    grad = ac.op("reduce_sum", (grad,), (i,), {"keep_dim": True, "dim_arg": [i]})
```

Comment:
Not really related to this PR - would it make sense to refactor code for "add", "subtract", "multiply"? The 'for' loops (when shapes differ) look the same for each op. @nvukobratTT @pilkicTT

Reply:
Personally, I don't see much value in it, but I would be fine with it if you wanted to... 😄

Reply:
Yeah, for now, let's keep it separated for flexibility. We can think about a better structure once we get at least a few models running with valid graphs. 

However I agree, we can create a more unified functions to encapsulate same (or similar) bwd definitions.